tests/eas/load_tracking.py

```diff
@@ -537,6 +540,206 @@ def test_load_avg_behaviour(self, experiment, tasks):
+            calibration=test_env.calibration())
+        return rtapp.conf(kind='profile',
+            params={'task'+str(i): task.get() for i, task in enumerate(tasks)},
+            run_dir=test_env.target.working_directory+'/run_dir/')
+
+    @classmethod
+    def get_wload(cls, json_file, tasks_nb):
+        """
+        Get a specification for a rt-app workload from a json file
+        :param json_file: path to the file that contains the tasks description
+        :type json_file: str
+
+        :param tasks_nb: number of tasks contained by the file
+        :type tasks_nb: int
+        """
+
+        return {
+            'migration': {
+                'type' : 'rt-app',
+                'conf' : {
+                    'class' : 'custom',
+                    'json' : json_file,
+                    'tasks' : tasks_nb,
+                    'prefix' : 'mig_test',
+                },
+            }
+        }
+
+    def _get_util_mean(self, start, end, df, cpus):
+        """
+        Compute the mean utilization per CPU given a time interval
+        :param start: start of the time interval
+        :type start: float
+
+        :param end: end of the time interval
+        :type end: float
+
+        :param df: data frame of the sched_load_cfs_rq event
+        :type df: DataFrame
+
+        :param cpus: list of CPUs for which the mean utilization needs to be
+                     computed
+        :type: cpus: [int]
+        :return a list containing the mean utilization for each CPU
+        """
+        util_mean = []
+        df = df[start:end]
+        for cpu in cpus:
+            util_per_cpu = df[(df.cpu == cpu) & (df.path == '/')].util
+            util_mean.append(area_under_curve(util_per_cpu) / (end - start))
+
+        return util_mean
+
+    def _get_util_expected(self, tasks, cpus, phase):
+        """
+        Compute the expected utilization per CPU given a phase.
+        :param tasks: json description of the tasks
+        :type tasks: dict
+
+        :param cpus: list of CPUs for which the expected utilization needs to
+                     be computed
+        :type cpus: [int]
+
+        :param phase: name of the phase for which the expected utilization
+                      needs to be computed
+        :type phase: str
+        :return a list containing the expected utilization for each CPU
+        """
+        expected = []
+        for cpu in cpus:
+            expected_val = 0
+            for task in tasks.iteritems():
+                period_us = task[1]['phases'][phase]['timer']['period']
+                run_time_us = task[1]['phases'][phase]['run']
+                duty_cycle = run_time_us * 100 / period_us
+                [phase_cpu] = task[1]['phases'][phase]['cpus']
+                expected_val += 1024 * (duty_cycle / 100.0) \
+                                if phase_cpu == cpu else 0
+            expected.append(expected_val)
+
+        return expected
+
+    def _get_cpus(self, tasks):
```

Comment:
+1, that's exactly what I'm proposing also above for the other params

Reply:
Ok